playground/components/header.tsx

```diff
@@ -61,88 +71,109 @@ export const Header: Component<{
               </svg>
-            }
-          >
-            <svg class="h-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
-              <path
-                stroke-linecap="round"
-                stroke-linejoin="round"
-                stroke-width="2"
-                d="M12 3v1m0 16v1m9-9h-1M4 12H3m15.364 6.364l-.707-.707M6.343 6.343l-.707-.707m12.728 0l-.707.707M6.343 17.657l-.707.707M16 12a4 4 0 11-8 0 4 4 0 018 0z"
-              />
-            </svg>
-          </Show>
-        </button>
-
-        <label
-          class="px-3 py-2 focus:outline-none focus:ring-1 rounded text-white opacity-80 hover:opacity-100 cursor-pointer"
-          title="Import from JSON"
-        >
-          <input type="file" class="sr-only" onChange={uploadFile} accept=".json" />
-          <span class="sr-only">Import from JSON</span>
-          <Icon path={upload} class="h-6" />
-        </label>
-
+              <span class="text-xs md:sr-only">Export to JSON</span>
+            </button>
+
+            <button
+              type="button"
+              onClick={() => exportToCsb(props.tabs)}
+              class="flex flex-row space-x-1 items-center px-3 py-2 focus:outline-none focus:ring-1 rounded text-white opacity-80 hover:opacity-100"
+              title="Export to CodeSandbox"
+            >
+              <svg class="fill-current h-6" style={{ padding: "0px 1.63px 0px 1.63px" }} preserveAspectRatio="xMidYMid" viewBox="0 0 256 296">
+                <path d="M115 261V154l-91-52v61l42 24v46l49 28zm24 1l51-29v-47l42-25v-60l-93 54v107zm81-181l-49-28-43 24-43-24-49 28 92 53 92-53zM0 222V74L128 0l128 74v148l-128 74L0 222z" />
+              </svg>
+              <span class="text-xs md:sr-only">Export to CodeSandbox</span>
+            </button>
+
+            <button
+              type="button"
+              onClick={shareLink}
+              class="flex flex-row space-x-1 items-center px-3 py-2 focus:outline-none focus:ring-1 rounded"
+              classList={{
+                'text-white opacity-80 hover:opacity-100': !copy(),
+                'text-green-100': copy(),
+              }}
+              title="Share with a minified link"
+            >
+              <Icon class="h-6" path={copy() ? link : share} />
+              <span class="text-xs md:sr-only">{copy() ? 'Copied to clipboard' : 'Share'}</span>
+            </button>
+          </div>
+        </div>
         <button
           type="button"
-          onClick={() => exportToJSON(props.tabs)}
-          class="px-3 py-2 focus:outline-none focus:ring-1 rounded text-white opacity-80 hover:opacity-100"
-          title="Export to JSON"
+          id="menu-btn"
+          class="z-40 px-3 py-2 focus:outline-none focus:ring-1 rounded text-white opacity-80 hover:opacity-100 visible relative md:hidden"
+          style={{margin:"0 10px 0 0"}} //adjust button to the right to better algin menu
```

Comment:
This can be written as follow, although I generally prefer only margin on the left and top:
```suggestion
          class="z-40 px-3 py-2 focus:outline-none focus:ring-1 rounded text-white opacity-80 hover:opacity-100 visible relative md:hidden m-0 mr-[10px]"
```